snaps/features/custom-ui/index.md

```diff
@@ -1126,6 +1273,51 @@ await snap.request({
 <img src={require("../../assets/custom-ui-tooltip.png").default} alt="Tooltip UI example" width="450px" style={{border: "1px solid #DCDCDC"}} />
 </p>
 
+### `Value`
+
+Outputs a component that displays two text values side by side. This component can only be used as a child of the [`Row`](#row) component.
```

Comment:
Can you update the `children` prop of the `Row` component to include `Value` as an option?